parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -549,10 +549,12 @@ pub mod pallet {
 			);
 
 			// Checks that we don't skip execution headers, they need to be imported sequentially.
+			let compact_execution_header: CompactExecutionHeader =
```

Comment:
The changes you made here in `process_execution_header_update` seem unnecessary. I'd like to reduce the amount of changes if at all possible

Reply:
Clean in https://github.com/Snowfork/snowbridge/pull/981/commits/3580c4de7d6400880f3a4df171ffb42b5000737b